docs/devices/EFR32MG21.Router.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Custom devices (DiY) EFR32MG21.Router control via MQTT"
+description: "Integrate your Custom devices (DiY) EFR32MG21.Router via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
+addedAt: 2022-10-26T20:00:00Z
+pageClass: device-page
+---
+
+<!-- !!!! -->
+<!-- ATTENTION: This file is auto-generated through docgen! -->
+<!-- You can only edit the "Notes"-Section between the two comment lines "Notes BEGIN" and "Notes END". -->
+<!-- Do not use h1 or h2 heading within "## Notes"-Section. -->
+<!-- !!!! -->
+
+# Custom devices (DiY) CC2530.ROUTER
+
+|     |     |
+|-----|-----|
+| Model | EFR32MG21.Router  |
+| Vendor  | Custom devices (DiY)  |
+| Description | EFR32MG21 router |
+| Exposes | linkquality |
+| Picture | ![Custom devices (DiY) EFR32MG21.Router](https://www.zigbee2mqtt.io/images/devices/EFR32MG21.Router.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+
+### LED status
+
+RED LED off = NO Netwrok
+RED LED blink = steering network
```

Comment:
Steering is a difficult term, can you explain it better?

Reply:
Changed the wording